q2_fondue/utils.py

```diff
@@ -24,27 +25,44 @@ class DownloadError(Exception):
     pass
 
 
+def _chunker(seq, size):
+    # source: https://stackoverflow.com/a/434328/579416
+    return (seq[pos:pos + size] for pos in range(0, len(seq), size))
+
+
 def _validate_esearch_result(
-        esearcher: es.Esearcher, run_ids: List[str], log_level: str) -> dict:
+        email: str, n_jobs: int, run_ids: List[str], log_level: str) -> dict:
     """Validates provided accession IDs using ESearch.
```

Comment:
Now that you moved the whole Esearcher into this function, I'd suggest to rename it to `_validate_run_ids` as that's what it is actually doing now and there is no "esearch_result" from the point of view of the upstream functions that would be calling this one.